dev_scripts/dangerzone-image

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
```

Comment:
I'm not sure this file is required altogether, so maybe remove it and use the dangerzone-image command instead?

Reply:
Checked and it's actually required, otherwise the share folder (containing cosign and the keys) is not resolved correctly on the dev machine. We should find a better solution for this in the long run that playing with `sys.dangerzone_dev` but that's how it's done at the moment, and orthogonal to this PR.